lighthouse-core/audits/dobetterweb/no-console-time.js

```diff
@@ -60,17 +60,7 @@ class NoConsoleTimeAudit extends Audit {
     const pageHost = new URL(artifacts.URL.finalUrl).host;
     // Filter usage from other hosts and keep eval'd code.
     const results = artifacts.ConsoleTimeUsage.usage.filter(err => {
-      let result = err.url;
-
-      if (!err.isEval) {
-        try {
-          result = new URL(err.url).host === pageHost;
-        } catch (err) {
-          result = false;
-        }
-      }
-
-      return result;
+      return err.isEval ? err.url : new URL(err.url).host === pageHost;
```

Comment:
shouldn't this be `!!err.url` to also be boolean? @ebidel since this was in the original